README.md

```diff
@@ -39,11 +39,11 @@
 
 ### Docker
 
-If you have docker installed simply build the Dockerfile and run the docker-compose.yml  
+If you have docker installed simply build the Dockerfile and run the docker-compose.yml as an admin/sudo user.
 files mentioned above need to be prepared as explained however
 ```bash
 $ docker build -t pouty-bot .
-$ docker compose up
+$ docker-compose up
```

Comment:
newer installations of docker have compose as subcommand 
so this was correct before